### DISCUSSION TITLE
SMPlayer and Dolby Atmos

### DISCUSSION THREAD

**Discussion by HikaruGunner** [smplayer-dev/smplayer#603] [Q&A]
Can SMPlayer play files in Dolby Atmos? I have some files encoded in dolby atmos that work fine in player like VLC (dolby atmos logo appears on my soundbar when a compatible file is playing), but they only play in STEREO on SMPlayer. Dolby Digital files play just fine though on all players.

**Comment by bob860712** [comment]
Enable General>Audio>AC3/DTS passthrough over S/PDIF and HDMI.
Channels by default: Default.
Then go to Preferences>Advanced>Mplayer/mpv>Options, and type in       -audio-spdif=dts-hd,truehd,eac3,ac3
